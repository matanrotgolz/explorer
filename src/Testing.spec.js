import getWheater from './Components/WeatherData';

describe('trying to fetch WeatherData',()=>{
    test('Should fetch Weather Data based on  location',()=>{
        const output ={
            'getWeather': !null
        };
        expect(getWheater).toEqual(output);
    })
})