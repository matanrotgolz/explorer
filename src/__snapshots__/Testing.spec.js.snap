// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test Link changes the class when hovered 1`] = `
<a
  page="http://www.facebook.com"
>
  Facebook
</a>
`;
